src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -957,7 +969,7 @@ internal static Attribute[] ReflectGetAttributes(Type type)
                 {
                     // Get the type's attributes.
                     //
-                    attrs = type.GetTypeInfo().GetCustomAttributes(typeof(Attribute), false).ToArray();
+                    attrs = type.GetTypeInfo().GetCustomAttributes(typeof(Attribute), false).OfType<Attribute>().ToArray();
```

Comment:
I just checked the reference assembly for System.Runtime. MemberInfo implements System.Reflection.ICustomAttributeProvider which has the GetCustomAttributes method that returns an object[].
The extension method GetCustomAttributes in CustomAttributeExtensions returns an IEnumerable<Attribute> and has the same signature.